html/captive-portal/lib/captiveportal/PacketFence/Controller/Signup.pm

```diff
@@ -386,6 +386,11 @@ sub doSmsSelfRegistration : Private {
     my $mobileprovider = $request->param("mobileprovider");
     my ($pid, $phone)  = @{$session}{qw(guest_pid phone)};
 
+    if ($self->reached_retry_limit($c, 'sms_request_retries', $portalSession->profile->{_sms_request_retries})) {
+        $logger->info("Max tries reached for requesting sms for $mac");
```

Comment:
I would rephrase it like this: *Maximum number of SMS signup requests reached for $mac*